### ISSUE TITLE
Homebridge 1.6 - not working

### ISSUE THREAD

**Issue by korgoth** [arachnetech/homebridge-mqttthing#609]
Hey guys - is anyone having an issue after updating to homebridge 1.6? 

Plugin just seizes to work. None of the accessories respond or take any action. I see the zigbee things being populated to the MQTT server so its not an issue on that end - it seems to be plugin related, but i cant see anything in the logs. 

After reverting to Homebridge 1.5.1 - everything went back to normal.

**Comment by korgoth** [comment]
Hasnt anyone updated HB to 1.6?

**Comment by jbuttnz** [comment]
Did you resolve this? I'm having problems too.

**Comment by korgoth** [comment]
No! I’m still stuck on HB 1.5.1 because of this issue and i am not able to find any information anywhere

**Comment by arachnetech** [comment]
I've not tried 1.6 yet, but I'll try updating my development environment and see what happens.

(I now mostly use Home Assistant for my own home automation now, although I still use Homebridge and MQTT-Thing for my iOS integration, and to support some custom components.)

**Comment by arachnetech** [comment]
My test configs are running ok with Homebridge 1.6.

**Comment by korgoth** [comment]
Very strange. Today i saw an error in the zigbee2mqtt staartup logs that stated something like: 
 Error: Struct length mismatch (expected=16/16, got=32)
 
 I updated all the dependancies for zigbee2mqtt, upgraded my mosquitto installation and still no luck. 
 
 The zigbee2mqtt process starts without any errors or warnings now. I can see my devices join the network and i can see them sending messages to the zigbee dongle and form there i see them appear in the mosquitto. Still homekit-mqttthing does not work. 
 
 Whenever i revert back to 1.5.1 it works without any problem. I cant even figure out where to look next as all the logs are clean..

**Comment by jeskko** [comment]
I got the same breakage upon updating to 1.6.1 and while investigating what exact accessory is causing it found that I had multiple duplicates of the same mqttthing -accessory in the config file. After cleaning the configuration and removing the duplicates, everything seems to work again. Strange that this didn't break anything on 1.5.x :)
